Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Use the official Golang image to build the Go binary
+FROM golang:1.22 AS builder
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+# Copy go.mod and go.sum files and download dependencies
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Copy the source code into the container
+COPY . .
+
+# Build the Go app
+RUN GOOS=linux GOARCH=amd64 go build -o main cmd/productivite/main.go
+
+# Use a minimal base image to run the Go binary
+FROM alpine:latest
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/8#issuecomment-2421530834>#5d7395</a><span> - 10/18/2024, 12:05 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The Alpine Linux version in the runtime image is not pinned to a specific version, which may lead to inconsistencies and potential security issues.
</div>
<div id="fix">
<b>Fix:</b> Pin the Alpine Linux version to a specific version to ensure consistency and security in the runtime environment.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -17,2 +17,2 @@
  # Use a minimal base image to run the Go binary
 -FROM alpine:latest
 +FROM alpine:3.18
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged